tests/x509/test_x509_ext.py

```diff
@@ -5208,3 +5208,11 @@ def test_hash(self):
         nonce3 = x509.OCSPNonce(b"1" * 5)
         assert hash(nonce1) == hash(nonce2)
         assert hash(nonce1) != hash(nonce3)
+
+
+class TestExtensionOIDNameDefined(object):
+    def test_all_extension_oid_members_have_names_defined(self):
+        public_members = [x for x in dir(ExtensionOID) if x[:2] != '__']
```

Comment:
Let's just rewrite these two loops as a for loop that does the assertion inside it. e.g.

```python
for oid in dir(ExtensionOID):
    if oid[:2] == '__':
        continue
    assert getattr(ExtensionOID, oid) in _OID_NAMES
```